File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
class Solution {
    public int solution(int[][] dots) {
        int length1 = 0;
        int length2 = 0;
        
        for(int i=1; i<dots.length; i++){
            int temp1 = dots[0][0];
            if(temp1 != dots[i][0]){
                length1 = Math.abs(dots[0][0] - dots[i][0]);
            }
            
            int temp2 = dots[0][1];
            if(temp2 != dots[i][1]){
                length2 = Math.abs(dots[0][1] - dots[i][1]);
            }
        }

        return length1 * length2;
    }
}

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    num1 = max(abs(dots[0][0]-dots[1][0]) , abs(dots[0][0]-dots[2][0]))
    num2 = max(abs(dots[0][1]-dots[1][1]) , abs(dots[0][1]-dots[2][1]))
    return num1 * num2
    
